BUILD.md

```diff
@@ -23,5 +23,23 @@ To find the jar once it's built, just look at the last few lines of output from
 If you're going to make changes to the Scala code, you may want to use IntelliJ and it's Scala
 plugin to help with the Scala syntax...!
 
+If you use [Eclipse IDE](http://www.eclipse.org/), you can set-up your development environment by following these instructions:
+
+* Install `sbt` and build as-above
+* Install [Scala IDE for Eclipse](http://scala-ide.org/) into your Eclipse installation if not already installed
+* Add the `sbteclipse-plugin` to your set of local sbt plugins:
+
+```
+mkdir -p ~/.sbt/0.13/plugins && tee ~/.sbt/0.13/plugins/plugins.sbt <<EOF
```

Comment:
This was more specific than I had in mind - my personal preference would just be to provide a link to https://github.com/typesafehub/sbteclipse#for-sbt-013-and-up (with a note saying 'use the global config')  - that way the install instructions are less likely to get out of date! No biggie tho'